packages/world/src/systems/LogoffSystem.sol

```diff
@@ -15,16 +15,15 @@ import { ActionType } from "../codegen/common.sol";
 import { MIN_TIME_BEFORE_AUTO_LOGOFF } from "../Constants.sol";
 import { AirObjectID, PlayerObjectID } from "../ObjectTypeIds.sol";
 import { getUniqueEntity } from "../Utils.sol";
-import { requireValidPlayer } from "../utils/PlayerUtils.sol";
+import { requireValidPlayer, deletePlayer } from "../utils/PlayerUtils.sol";
 import { notify, LogoffNotifData } from "../utils/NotifUtils.sol";
 
 import { EntityId } from "../EntityId.sol";
 
 contract LogoffSystem is System {
   function logoffCommon(EntityId playerEntityId, VoxelCoord memory playerCoord) internal {
     LastKnownPosition._set(playerEntityId, playerCoord.x, playerCoord.y, playerCoord.z);
```

Comment:
should setting LastKnownPosition be inside deletePlayer?

Reply:
I'm thinking of reusing `deletePlayer` on death, and in that case don't need to set it